src/app/features/quest/quest-template/quest-template.component.html

```diff
@@ -402,6 +402,17 @@
             <input [formControlName]="'RewardMoney'" id="RewardMoney" type="number" class="form-control form-control-sm" />
           </div>
 
+          <div class="form-group col-12 col-sm-6 col-md-5 col-lg-4 col-xl-2">
+            <label class="control-label" for="RewardMoneyDifficulty">Rew.MoneyDifficulty</label>
+            <i class="fas fa-info-circle ms-1" [placement]="'auto'" [tooltip]="'TODO: RewardMoneyDifficulty tooltip'"></i>
+            <input
+              [formControlName]="'RewardMoneyDifficulty'"
```

Comment:
```suggestion
              formControlName="RewardMoneyDifficulty"
```

Reply:
probably you can do similarly the same for placement and tooltip (not 100% sure)

Reply:
I'd keep it like it is now to be consistent with the rest..